projects/laji/src/app/+theme/kerttu/directive/requires-audio-support.directive.ts

```diff
@@ -18,11 +23,21 @@ export class RequiresAudioSupportDirective implements OnInit {
       const factory = this.resolver.resolveComponentFactory(AudioNotSupportedErrorComponent);
       this.viewContainer.createComponent(factory);
     } else {
+      if (this.isIOS()) {
+        const factory = this.resolver.resolveComponentFactory(AudioIosWarningComponent);
+        this.viewContainer.createComponent(factory);
+      }
       this.viewContainer.createEmbeddedView(this.templateRef);
     }
   }
 
   private webAudioApiIsSupported() {
     return !!this.audioService.audioContext;
   }
+
+  private isIOS() {
```

Comment:
This could be moved to PlatformService where we already have few other checks like this awailable (same for the webAudioApiIsSupported method)